transforms/universal/doc_id/kfp_ray/doc_id_wf.py

```diff
@@ -22,16 +22,16 @@
 # the name of the job script
 EXEC_SCRIPT_NAME: str = "doc_id_transform_ray.py"
 # components
-base_kfp_image = "quay.io/dataprep1/data-prep-kit/kfp-data-processing:latest"
+base_kfp_image = "quay.io/dataprep1/data-prep-kit/kfp-data-processing_v2:latest"
```

Comment:
No need to change the default image name

Reply:
Fixed.